x/commitment/module.go

```diff
@@ -123,7 +123,7 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
 	m := migrations.NewMigrator(am.keeper)
-	err := cfg.RegisterMigration(types.ModuleName, 10, m.V11Migration)
+	err := cfg.RegisterMigration(types.ModuleName, 11, m.V12Migration)
```

Comment:
keep v11Migration as well otherwise in mainet it won't happen and upgrade will fail. (as on mainnet first migration happened yet)

Reply:
added, thanks